Dockerfile

```diff
@@ -33,11 +33,12 @@ RUN apt-get update && apt-get install -y apt-utils && apt-get -y dist-upgrade &&
 
 RUN ln -s /usr/bin/python${PYTHON_VER} /usr/bin/python
 
-RUN pip install --no-cache-dir torch==1.7.1+cu110 torchvision==0.8.2+cu110 -f https://download.pytorch.org/whl/torch_stable.html
+RUN pip install --no-cache-dir torch==1.9.0+cu111 torchvision==0.10.0+cu111 torchaudio==0.9.0 -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
Not sure why these are showing up as changed. Torch 1.9 should be in master already